src/components/namestable/worker-edit.helper.tsx

```diff
@@ -0,0 +1,17 @@
+import {
+  WorkerTypeHelper,
+  WorkerType,
+  ContractType,
+  ContractTypeHelper,
+} from "../../common-models/worker-info.model";
+import { StringHelper } from "../../helpers/string.helper";
+
+export function translateAndCapitalizeWorkerType(workerType: WorkerType): string {
```

Comment:
Nie umieszczamy helperów  w komponentach. Jeśli kod jest wykorzystywany w jednym komponencie to dodajemy go tylko tam. Jeśli może pojawić się potrzeba ponownego wykorzystania to należy to wrzucić do folderu jakiegoś istniejącego helpera z helpers albo dodać nowego. 
Dodatkowo ten kod dla translateAndCapitalizeWorkerType oraz translateAndCapitalizeContractType  jest powtórzony. Można stworzyć jedną generyczną funkcję - translateAndCapitalize, która przyjmuje 2 parametry: pierwszy to helper, a drugi coś co helper może przetłumaczyć.